.github/workflows/ci.yml

```diff
@@ -338,7 +338,7 @@ jobs:
           node_selector: cloud.google.com/gke-nodepool=gha
           size: 30
           bootstrap: 10m
-          level: debug
+          level: info
```

Comment:
can you keep this as debug please.

Reply:
I want to check if reducing log level helps with the timeouts.

Reply:
actually never mind. i am fine with info for this test.